### DISCUSSION TITLE
Welcome to guake Discussions!

### DISCUSSION THREAD

**Discussion by gsemet** [Guake/guake#1888] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Davidy22** [comment]
Ooh nice, we got this enabled. I just use this program, and I had some stuff I wanted to add

**Reply by gsemet** [reply]
Yes, let’s see what we can do with it
